openupgrade_scripts/scripts/base/17.0.1.3/post-migration.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Viindoo Technology Joint Stock Company (Viindoo)
+# Copyright 2023 Hunki Enterprises BV (https://hunki-enterprises.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_deleted_xml_records = [
+    "base.icp_mail_bounce_alias",
+    "base.icp_mail_catchall_alias",
+    "base.icp_mail_default_from",
```

Comment:
These three pieces of data seem to have been deleted during the pre-migration phase.

Reply:
Are you sure ?, normally those xml-id that have noupdate need to be manually deleted using migration script, in lower version (15, 16) i usually do that

Reply:
> Are you sure ?, normally those xml-id that have noupdate need to be manually deleted using migration script, in lower version (15, 16) i usually do that


Sorry, it seems I didn’t express myself clearly before; What I meant was that the three _deleted_xml_records and their corresponding deletion methods appear repeatedly in post_migration.py and end_migration.py. I think it’s only necessary to clean them up once in either place.